CHANGELOG.md

```diff
@@ -12,6 +12,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 -   Fee Recovery checkbox placement in Multi-Step forms now respects the Fee Recovery input location setting (#5205)
 -   FFM inputs are now setup on init of the Multi-Step form, to ensure they work with only a single gateway enabled (#5216)
 
+### Added
+
+-   Milestone block is now available in the block editor (#5224)
```

Comment:
@henryholtgeerts Can you pull latest `develop` into this branch? This change you just added should be the only unreleased change in the changelog.

Reply:
Done, ref: https://github.com/impress-org/givewp/pull/5224/commits/4ada6e722e7413ad42467f553b52201d0b377e26